homeassistant/components/zwave/manifest.json

```diff
@@ -4,6 +4,5 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/zwave",
   "requirements": ["homeassistant-pyozw==0.1.10", "pydispatcher==2.0.5"],
-  "after_dependencies": ["ozw"],
```

Comment:
The migration from zwave to ozw needs the other integration.

Reply:
So, what if you are running both?  Like myself?

Reply:
And also...  If the official path is ZWaveJS, why does this migration even exist?

Reply:
Does this cause an issue if someone has `zwave` but not `ozw`?